src/Microsoft.Extensions.WebEncoders/EncoderServiceCollectionExtensions.cs

```diff
@@ -17,14 +20,23 @@ public static class EncoderServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection"/>.</param>
         /// <returns>The <see cref="IServiceCollection"/> instance after the encoders have been added.</returns>
-        public static IServiceCollection AddWebEncoders(this IServiceCollection services)
+        public static void AddWebEncoders(this IServiceCollection services)
```

Comment:
That's a shame, the chaining was nice, couldn't we require all of our builders to implement IServiceCollection to keep the chaining intact?